### ISSUE TITLE
Error when using registerEnumType from other module

### ISSUE THREAD

**Issue by georgyfarniev** [19majkel94/type-graphql#387]
Hello. Thank you for your help for previous issue, now after hoisting in lerna I can use models in different modules. But I have another problem: when I use registerEnumType function in my core library and then consume this type in application, I got the following error:

`TypeError: Cannot convert undefined or null to object`

I do everything as documented for enum. What could be a problem in this case? Thanks in advance.

**Comment by 19majkel94** [comment]
How do you export the registered enum from your module? Can you show a code snippet?

Can you console.log the imported enum where you use it as a field type?

**Comment by georgyfarniev** [comment]
@19majkel94 , this is how I export enum:

```js
enum Something {
  Key1 = 'Value1',
  Key2 = 'Value2'
}

registerEnumType(Something, {
  name: 'Something',
  description: 'Some enum'
})

export { Something }
```


This is what I get when execute `console.dir` in importing application code:
```js
{
  Key1: 'Value1',
  Key2: 'Value2',
}
```

Do you have any idea? Also, I will try to reproduce it later in repository with minimal environment that I created before.

**Comment by 19majkel94** [comment]
> Do you have any idea?

Not really 😕 Minimal repo and full error with stacktrace would be helpful.

**Comment by georgyfarniev** [comment]
@19majkel94 I wasn't able to repro in isolated environment. I will close it for now and will reopen if it will make sense after investigation.

**Comment by deweve** [comment]
I've got the error to, I was declaring the enum in a file and register the Enum type in another file where I import the enum. I've just simply moved the registration of the enum in the same file where the enum is declared. However I do not know why there is a problem when I'm registering an enum that have not been declared in the same file. Maybe the documentation could mention this problem if it is a known problem.

PS: This framework is great
